api/config/shared/global.proto

```diff
@@ -100,8 +100,26 @@ message External {
 		SSL ssl = 5;
 	}
 
+	message Automate {
+		message Authentication {
+			google.protobuf.StringValue scheme = 1;
+			google.protobuf.StringValue token = 2;
+		};
+
+		message SSL {
+			google.protobuf.StringValue root_cert = 1;
+			google.protobuf.StringValue server_name = 2;
+		};
+
+		google.protobuf.BoolValue enable = 1;
+		google.protobuf.StringValue node = 2;
```

Comment:
I'm not sure I like `node` here as we treat it as an endpoint url. Perhaps something like `endpoint` or `url` would be better?

Reply:
We called things node for elasticsearch and postgres